docker-compose.override.yml

```diff
@@ -15,6 +15,7 @@ services:
     image: mysql:5.7
     volumes:
       - ./docker_data/mysql:/var/lib/mysql
+      - ./db-schema.sql:/docker-entrypoint-initdb.d/db-schema.sql
```

Comment:
I didn't know about this. So simply copy it into this location in the container and voila?! Very cool.

Reply:
Yes, it's one of the features of the MySQL image. It's described in [the "Initializing a fresh instance" on the image's Docker Hub page](https://hub.docker.com/_/mysql). The PostgreSQL image has something similar.